vignettes/introduction.Rmd

```diff
@@ -241,11 +241,12 @@ we think the defaults are a good starting point [@topcuoglu_framework_2020].
 
 When `training_frac` is a fraction between 0 and 1, a random sample of
 observations in the dataset are chosen for the training set to satisfy the
-`training_frac`. However, in some cases you might wish to control exactly which
-observations are in the training set. You can instead assign `training_frac` a
-vector of indices that correspond to which rows of the dataset should go in the
-training set (all remaining sequences will go in the testing set).
-Here's an example with ~80% of the data in the training set:
+`training_frac` using `get_partition_indices()`. However, in some cases you
+might wish to control exactly which observations are in the training set. You
+can instead assign `training_frac` a vector of indices that correspond to which
+rows of the dataset should go in the training set (all remaining sequences will
+go in the testing set). Here's an example with ~80% of the data in the training
```

Comment:
here it says the example has ~80% of data in training set but below in the code chunk training frac is set to 0.7?

Reply:
@courtneyarmour I think you're looking at the code chunk at L417, but the chunk this refers to at L252 does use 0.8.

Reply:
ah right missed that there was a chunk of code hidden!